enterprise/backend/src/metabase_enterprise/workspaces/common.clj

```diff
@@ -13,13 +12,6 @@
    [metabase.util.quick-task :as quick-task]
    [toucan2.core :as t2]))
 
-(defn mock-mapping
-  "This is a mocked mapping for execution, used until we get real mapping integrated."
-  [ws target]
-  (assoc target
-         :schema (ws.u/isolation-namespace-name ws)
-         :name (ws.u/isolated-table-name target)))
-
```

Comment:
Now it makes sense why you were having trouble finding where to put this - it didn't belong in this world!